roles/content/templates/nginx.conf.j2

```diff
@@ -2,6 +2,7 @@ location / {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header Host $http_host;
     proxy_redirect off;
+    resolver 172.31.0.2 valid=30s;
```

Comment:
That magic IP is +2 to the bottom of the vpc we're using in us-west-2. I should have parameterized this, but didn't.

Reply:
https://ryangilliganblog.wordpress.com/2018/09/19/aws-nginx-returns-499-response-errors-when-using-downstream-load-balancer/